Grunt process breaks on macOS

macOS sometimes uses a file named `.DS_Store` to store metadata about the folder's content. This breaks the Grunt process as it is being treated as a schema, if it's created in the `schemas/json` folder. The error returned is:

```
Warning: Unable to read "schemas/json/.DS.Store.json" file (Error code: ENOENT). Use --force to continue.

Aborted due to warnings.
npm ERR! code ELIFECYCLE
npm ERR! errno 3
npm ERR! schemastore.org@1.0.0 build: grunt
npm ERR! Exit status 3
npm ERR!
npm ERR! Failed at the schemastore.org@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/alabetski/.npm/_logs/2020-08-05T11_53_23_200Z-debug.log
make: *** [build] Error 3
```

I am working on a fix for it.

--- a/src/Gruntfile.js
+++ b/src/Gruntfile.js
@@ -146,6 +146,11 @@ module.exports = function (grunt) {
         return;
       }
 
+      // If it's the .DS_Store folder from macOS, ignore and continue.
+      if (folder == ".DS_Store") {
+        return;
+      }
+
       const toTestFilePath = toPath(testDir, folder);
 
       const name = folder.replace("_", ".");

